go/runtime/host/sgx/sgx.go

```diff
@@ -136,7 +141,19 @@ func (ts *teeState) update(ctx context.Context, sp *sgxProvisioner, conn protoco
 	if ts.impl == nil {
 		return nil, fmt.Errorf("not initialized")
 	}
-	return ts.impl.Update(ctx, sp, conn, report, nonce)
+
+	retB, retE := ts.impl.Update(ctx, sp, conn, report, nonce)
+
+	if metrics.Enabled() {
+		teeAttestationsPerformed.With(prometheus.Labels{"runtime": ts.runtimeID.String()}).Inc()
+		if retE != nil {
+			teeAttestationsFailed.With(prometheus.Labels{"runtime": ts.runtimeID.String(), "error": retE.Error()}).Inc()
```

Comment:
Do we need the error message or is `status=succeeded|failed` enough? If the error is unique on every failure (e.g. it contains a timestamp), we will create a lot of times series here.

Reply:
I agree that the error message is too much. If one needs the actual error message, it should be looked up in the node's logs.

Reply:
OK, I'll remove it :)